omero/sysadmins/unix/server-centos7-ice36.rst

```diff
@@ -119,6 +119,17 @@ Configure the database and the location of the data directory:
     :start-after: #end-copy-omeroscript
     :end-before: #end-step04
 
+Weaker ciphers like ADH are disabled by default in new versions of OpenSSL
+and TLS versions 1.0 and 1.1 have been dropped from JDK packages.
+In order to connect to an OMERO.server using any OMERO clients
+e.g. the Java Desktop client, the OMERO.web client or the CLI and import data, 
+you need to generate self-signed certificates by running:
```

Comment:
This might could use an `or-` statement about bringing your own certificate. That's possible by setting a configuration property but I don't think we have or could easily have an example for that though.

Reply:
I have centralised the text and added a note to the server-installation page.
Since we do not have an example about bringing own certificates. I have omitted it.